apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktService.kt

```diff
@@ -61,14 +66,45 @@ class AktivitetspliktService(
         return hentAktivitetspliktOppfolging(behandlingId)!!
     }
 
+    suspend fun hentAktivitetspliktDto(
+        sakId: Long,
+        bruker: BrukerTokenInfo,
+    ): AktivitetspliktDto {
+        val grunnlag = grunnlagKlient.hentGrunnlagForSak(sakId, bruker)
+        val avdoedDoedsdato =
+            requireNotNull(
+                grunnlag
+                    .hentAvdoede()
+                    .singleOrNull()
+                    ?.hentDoedsdato()
+                    ?.verdi,
+            ) {
+                "Kunne ikke hente ut avdødes dødsdato for sak med id=$sakId"
+            }
+        val sisteBehandling = behandlingService.hentSisteIverksatte(sakId)
+        val aktiviteter = sisteBehandling?.id?.let { hentAktiviteter(it) } ?: emptyList()
+
+        val aktivitetsgrad = listOfNotNull(aktivitetspliktAktivitetsgradDao.hentNyesteAktivitetsgrad(sakId))
+        val unntak = listOfNotNull(aktivitetspliktUnntakDao.hentNyesteUnntak(sakId))
+
+        return AktivitetspliktDto(
+            sakId = sakId,
+            avdoedDoedsmaaned = YearMonth.from(avdoedDoedsdato),
+            aktivitetsgrad = aktivitetsgrad.map { it.toDto() },
```

Comment:
I utgangspunktet så har man vel enten unntak _eller_ aktivitetsgrad, så andre steder er det løst med å kun returnere den nyeste av de, og ikke begge to.

Reply:
Ja, det er sant! Jeg endrer denne til å ha samme type logikk enn så lenge, så må vi nok se nærmere på det når vi ser på periodisering der man _kan_ ha begge.